src/Symfony/Component/Notifier/Bridge/Lox24/Lox24Transport.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Lox24;
+
+use Symfony\Component\Notifier\Exception\InvalidArgumentException;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\Exception\ClientExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\DecodingExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\RedirectionExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\ServerExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Andrei Lebedev <andrew.lebedev@gmail.com>
+ */
+final class Lox24Transport extends AbstractTransport
+{
+    protected const HOST = 'api.lox24.eu';
+
+    public function __construct(
+        private readonly string $user,
+        #[\SensitiveParameter] private readonly string $token,
+        private readonly string $from,
+        private readonly array $options = [],
+        ?HttpClientInterface $client = null,
+        ?EventDispatcherInterface $dispatcher = null,
+    ) {
+        parent::__construct($client, $dispatcher);
+    }
+
+    public function __toString(): string
+    {
+        $params = [
+            'from' => $this->from,
+            ...$this->options,
+        ];
+
+        $query = $params ? '?'.http_build_query($params) : '';
+
+        return sprintf('lox24://%s%s', $this->getEndpoint(), $query);
+    }
+
+    public function supports(MessageInterface $message): bool
+    {
+        return $message instanceof SmsMessage
+            && (null === $message->getOptions() || $message->getOptions() instanceof Lox24Options);
+    }
+
+    /**
+     * @throws RedirectionExceptionInterface
+     * @throws DecodingExceptionInterface
+     * @throws ClientExceptionInterface
+     * @throws TransportExceptionInterface
+     * @throws ServerExceptionInterface
+     */
+    protected function doSend(MessageInterface $message): SentMessage
+    {
+        if (!$message instanceof SmsMessage) {
+            throw new UnsupportedMessageTypeException(__CLASS__, SmsMessage::class, $message);
+        }
+
```

Comment:
Please check another bridge, we need to double check here if the options class, if provided, is instance of Lox24Options

Reply:
fixed

Reply:
I don't see the check for the used options class

Reply:
@OskarStark  please check

https://github.com/alebedev80/symfony-lox24-notifier/blob/7.1/src/Symfony/Component/Notifier/Bridge/Lox24/Lox24Transport.php#L74

Reply:
got it, thanks